### PR TITLE
Update install process and documentation

### LINKED ISSUE #14: Improve installation process and documentation
- [x] Remove references to branch `tsmith-load-test`. (I've confirmed that tests appear to work on `dcp-cli@master`.)
- [x] Update dependencies: `locustion==0.9` required to use `--run-time` flag specified in README, yet requirements.txt specifies `locustio==0.8.1`.
- [x] Specify required Python version: v3.6 should be used since requirement `gevent==1.3` does not support v3.7.
- [ ] Update Docker documentation: references nonexistent directory / repo `DSS-Scalability/`
- [ ] Update Docker documentation: Dockerfile relies on non-LTS Ubuntu Artful, which is EOL as of June 2018 (and therefore no longer functioning)
- [x] Format of $HOST is unclear

### PR DESCRIPTION
**Pull request by natanlao**
Closes #14.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,14 +44,15 @@ Preconfigured test are located in `./scale_tests`. Additional scale test should
...
-If  using Elasticbeanstock add the new requirements to `DSS-scalability/eb-locustio-sample/.ebextensions/setup.config`
+If using Elastic Beanstalk add the new requirements to `DSS-scalability/eb-locustio-sample/.ebextensions/setup.config`
 under `locust36`.
 
 
 ##### Update chalice requirements
 downloads the packages for aws lambda env
-docker run -it --volume=$PWD:/pp python:3.6 bash -c "pip download invokust locustio==0.8.1 gevent==1.2.2 git+git://github.com/HumanCellAtlas/dcp-cli@tsmith-load-testing#egg=hca --dest=/pp" 
+
+    docker run -it --volume=$PWD:/pp python:3.6 bash -c "pip download invokust locustio==0.8.1 gevent==1.2.2 git+git://github.com/HumanCellAtlas/dcp-cli@tsmith-load-testing#egg=hca --dest=/pp"
```
Comment (amarjandu) [inline README.md]: that url does not resolve, can you update it?

Comment (natanlao) [reply README.md]: It resolves for me:

```
> pip download invokust locustio==0.8.1 gevent==1.2.2 git+git://github.com/HumanCellAtlas/dcp-cli@tsmith-load-testing#egg=hca --dest=./pp
...
Successfully downloaded invokust locustio gevent hca numpy pyzmq boto3 flask requests msgpack-python six greenlet crcmod cryptography google-auth google-auth-oauthlib Jinja2 jsonpointer jsonschema tweak dcplib argcomplete commonmark docutils PyJWT retrying jmespath s3transfer botocore itsdangerous Werkzeug click certifi chardet idna urllib3 cffi asn1crypto cachetools pyasn1-modules rsa requests-oauthlib MarkupSafe puremagic crc32c future python-dateutil pycparser pyasn1 oauthlib argparse
You are using pip version 19.0.3, however version 19.1.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```

**Comment by amarjandu** [comment]
its it possible to change the URL and remove reference to trents PR? its a stale-non-merged branch

File: README.md [hunk 2]
```diff
@@ -32,10 +31,16 @@ To run using docker you:
...
 
 #### Running from CLI
-`$ locust -f ./scale_tests/upload_cloud.py --host=local:1234 --no-web --client=100 --hatch-rate=50 --run-time=10s --csv=./scale_tests/upload`
+
+Using Python 3.6:
+
+    $ pip install -r requirements.txt
+    $ locust -f ./scale_tests/upload_cloud.py --host=$HOST --no-web --client=100 --hatch-rate=50 --run-time=10s --csv=./scale_tests/upload
+
+Where `$HOST` is the base URL of the DSS endpoint you want to hit, e.g. `https://dss.dev.data.humancellatlas.org/v1/`. (The tests will look for `${HOST}swagger.yaml`.)
```
Comment (amarjandu) [inline README.md]: @DailyDreaming is there a convention for keeping the leading slash with the file name? or having it at the host level as a trailing slash?

Comment (natanlao) [reply README.md]: For reference, I am referring to this line: https://github.com/HumanCellAtlas/data-store-scale-test/blob/2223be80cc1cfa515dfcbbe147bf7def31359844/locustfiles/common/dsslocust.py#L201

**Pull request merged**